presto-main/src/main/java/com/facebook/presto/sql/planner/SymbolsExtractor.java

```diff
@@ -85,6 +93,13 @@ private SymbolsExtractor() {}
         return builder.build();
     }
 
+    public static List<Symbol> extractAll(RowExpression expression)
+    {
+        ImmutableList.Builder<Symbol> builder = ImmutableList.builder();
```

Comment:
If you change this to be a `VariableReferenceExpression` builder it will save me a few lines of code when I get to this part for symbol to variable refactoring, lol.

Reply:
I'm ok with that...